### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maciejspychala**

### CODE REVIEW

**Comment by maciejspychala** [comment]
@kamilpp

File: app/assets/stylesheets/stories.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the stories controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (kamilpp) [inline app/assets/stylesheets/stories.scss]: Comments, empty files

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'stories/index'
+
+  get 'stories/show'
+
+  root 'stories#index'
+  get 'stories/index' => 'stories#index'
+  get 'stories/:id' => 'stories#show', as: :story
```
Comment (kamilpp) [inline config/routes.rb]: Simply: scopes, resources, namespaces
Later move to API

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
 #
 # Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Story.create(url: 'https://duckduckgo.com', title: 'search engine', author: 'maciek')
+Story.create(url: 'https://git-scm.com/', title: 'version control system', author: 'maciek')
```
Comment (kamilpp) [inline db/seeds.rb]: !

File: app/models/story.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+class Story < ApplicationRecord
```
Comment (kamilpp) [inline app/models/story.rb]: Remember to always put validations.

File: Gemfile [hunk 5]
```diff
@@ -8,8 +8,8 @@ end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.4'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+# Use postgresql as the database for Active Record
+gem 'pg'
```
Comment (kamilpp) [inline Gemfile]: `rails new -d`

**Pull request merged**